Source/WebCore/platform/graphics/BitmapImage.cpp

```diff
@@ -393,7 +385,7 @@ bool BitmapImage::canUseAsyncDecodingForLargeImages() const
 
 bool BitmapImage::shouldUseAsyncDecodingForAnimatedImages() const
 {
-    return canAnimate() && m_allowAnimatedImageAsyncDecoding && (shouldUseAsyncDecodingForTesting() || m_source->canUseAsyncDecoding());
```

Comment:
Can you explain why we no longer need `m_allowAnimatedImageAsyncDecoding` and no longer consult `shouldUseAsyncDecodingForTesting()`?